lib/java/change-directory/src/main/java/org/enso/change/directory/WorkingDirectory.java

```diff
@@ -0,0 +1,9 @@
+package org.enso.change.directory;
+
+public interface WorkingDirectory {
```

Comment:
You don't want 3rd parties to implement this interface:
```suggestion
public abstract class WorkingDirectory {
    WorkingDirectory() {}
```